lisa/node.py

```diff
@@ -426,6 +426,20 @@ def get_working_path(self) -> PurePath:
 
         return self.get_pure_path(result.stdout)
 
+    def find_partition_with_freespace(self, size_in_gb: int) -> str:
+        df = self.tools[Df]
+        home_partition = df.get_partition_by_mountpoint("/home")
+        if home_partition and df.check_partition_size(home_partition, size_in_gb):
```

Comment:
Do we need this function at the node level? This seems very specific to Linux distros and might not be applicable to other platforms.

Reply:
Ah I didn't consider the portability issue. I do think it would be useful to have a helper function to get a disk with free space for Windows too, maybe we could switch based on node type? We don't need to implement Windows yet but it would leave the door open to do it later.

Reply:
Updated to check for node.os.is_windows